src/go/rpk/pkg/tuners/redpanda_checkers.go

```diff
@@ -237,5 +239,18 @@ func RedpandaCheckers(
 		ClockSource:                   []Checker{NewClockSourceChecker(fs)},
 		Swappiness:                    []Checker{NewSwappinessChecker(fs)},
 		KernelVersion:                 []Checker{NewKernelVersionChecker(GetKernelVersion)},
-	}, nil
+	}
+
+	v, err := cloud.AvailableVendor()
```

Comment:
do you think it may be an issue ? I mean checkers are created in separate command, start uses tuners where I pass the checker

Reply:
It \*might\* make startup time longer? But I'm not sure if it's perceivably longer.

Reply:
@mmaslankaprv you can test w/out a network. i bet pretty significant (multiple seconds) offline. 
I wonder if you should have a 1 second timeout.

Reply:
If so, it's worth adding the timeout logic to `cloud.AvailableVendor` itself, now that it will be used in more than 1 place: https://github.com/vectorizedio/redpanda/blob/ad638c84cd4578acb28cc51eed922c928d5119d7/src/go/rpk/pkg/cloud/vendors.go#L34